martin/src/srv/tiles_info.rs

```diff
@@ -58,6 +53,29 @@ async fn get_source_info(
     Ok(HttpResponse::Ok().json(merge_tilejson(&sources, tiles_url)))
 }
 
+fn generate_tiles_base(
+    srv_config: &SrvConfig,
+    req: &HttpRequest,
+    path: &Path<SourceIDsRequest>,
+) -> String {
+    let mut result = None;
+    if let Some(path_from_config) = &srv_config.base_path {
+        let base = format!("{path_from_config}/{0}", &path.source_ids);
```

Comment:
Should we allow the SrvConfig.base_path with a tail slash, like "/foo/bar/"?

Reply:
That's a good question... `X-Rewrite-URL` and `--base-path` must use exactly the same rules - so if one allows something, so should the other. We may allow some "normalization" logic (for both cases) - e.g. if it has a trailing slash, we should internally convert it to a string without it.

Reply:
Traling '/' would be removed in prase_base_path function now.